packages/lab/test/regressions/README.md

```diff
@@ -0,0 +1 @@
+Note: Not active, just a placeholder for the SpeedDial regression test.
```

Comment:
Should the regression tests be shared between different packages?

Reply:
???

Reply:
SpeedDial was originally written to go into material-ui/src, so a regression test was written. Rather than throw that code away, I've put it here. Nothing more, nothing less.